build/lib/layersChecker.js

```diff
@@ -50,7 +50,9 @@ const CORE_TYPES = [
     'self',
     'trimLeft',
     'trimRight',
-    'queueMicrotask'
+    'queueMicrotask',
```

Comment:
`,` at the end of the last item in an array could avoid it looking like there is a change to line 53, since this is only an insertion.